src/com/mambu/apisdk/services/AccountingService.java

```diff
@@ -125,4 +170,79 @@ public GLAccount getGLAccount(String glCode, String fromDate, String toDate) thr
 
 		return serviceExecutor.execute(getGLJournalEntries, params);
 	}
+
+	// Params definitions for posting GL Journal Entries. See MBU-1737
+	private static String debitAccountParam = "debitAccount";

```

Comment:
OK. I will move them to the APIData class (where all other simmer parameters are)